.gitignore

```diff
@@ -245,3 +245,4 @@ bh_unicode_properties.cache
 GitHub.sublime-settings
 
 # End of https://www.gitignore.io/api/sublimetext
+assets/package-lock.json
```

Comment:
Why remove package-lock.json from VCS?

Reply:
I found it to be adding a lot of noise to diffs, but I'll create a separate commit for it instead.